source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Publisher: Unprocessed fact-check emails'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-07-02
+review_in: 6 months
+---
+
+### Unprocessed fact-check emails
+
+This means there are unprocessed emails in the fact-check inbox
+(govuk-fact-check@digital.cabinet-office.gov.uk).
+Publisher has a [script (`mail_fetcher`)](script) that checks the inbox
+[every 5 minutes][schedule] and [processes all emails present][process].
+
+If emails are not processed or deleted, this alert is triggered. The level of
+the [alert][] depends on the number of emails present - which was chosen
+arbitrarily and is currently:
```

Comment:
You _may_ need a newline before the `-` on the next line (I know Trello struggles to recognise lists in markdown when this is the case - hopefully the dev docs are smarter!)

Reply:
You're right. We don't have a newline [here](https://github.com/alphagov/govuk-developer-docs/blob/master/source/manual/add-translation-whitehall.html.md#L11-L14) and it displays incorrectly [here](https://docs.publishing.service.gov.uk/manual/add-translation-whitehall.html):
![Screenshot 2020-07-03 at 11 11 47](https://user-images.githubusercontent.com/14851208/86459143-47d48500-bd1e-11ea-88ea-e56bb3e08a5b.png)
(I'll fix that in a separate PR)